python/ray/ml/BUILD

```diff
@@ -1,3 +1,57 @@
+# --------------------------------------------------------------------
+# Tests from the python/ray/ml/examples directory.
+# Please keep these sorted alphabetically.
+# --------------------------------------------------------------------
+py_test(
+    name = "custom_trainer",
+    size = "small",
+    main = "examples/custom_trainer.py",
+    srcs = ["examples/custom_trainer.py"],
+    tags = ["team:ml", "exclusive"],
+    deps = [":train_lib"]
```

Comment:
`:ml_lib`?

Reply:
Yes you are 100% correct, it should be ml_lib!

Reply:
Updated